collector/pkg/component/analyzer/network/gauge_pool.go

```diff
@@ -23,19 +23,24 @@ func createDataGroup() interface{} {
 	return dataGroup
 }
 
-type DataGroupPool struct {
+type DataGroupPool interface {
+	Get() *model.DataGroup
+	Free(dataGroup *model.DataGroup)
+}
+
+type GaugeDataGroupPool struct {
```

Comment:
We don't use `Gauge` as the name of data anymore. I think `SimpleDataGroupPool` is a better name.